tensorboard/webapp/widgets/custom_modal/custom_modal_component.ts

```diff
@@ -56,18 +56,20 @@ export class CustomModalComponent implements OnInit {
   @Output() onClose = new EventEmitter<void>();
 
   readonly visible$ = new BehaviorSubject(false);
+  private readonly canClose$ = new BehaviorSubject(true);
```

Comment:
Does this actually need to be a BehaviorSubject? Can it just be a plain old boolean?

Reply:
Yes, it could be a plain boolean. For some reason I was thinking this would be nice in the future but I'm not sure it's worth the added complexity.